### PR TITLE
[Docs site] Fixed failing builds?

### PR DESCRIPTION
**Pull request by kodster28**
cc: @angelampcosta and @victor-perov, think #6458 broke our builds

### CODE REVIEW

File: content/analytics/graphql-api/getting-started/execute-graphql-query.md [hunk 1]
```diff
@@ -37,7 +37,6 @@ you can see how `echo` piped together with `tr` to provide a proper payload with
 ---
 header: Example bash script that uses curl to query Analytics API
 ---
-```bash
```
Comment (kodster28) [inline content/analytics/graphql-api/getting-started/execute-graphql-query.md:40]: There is a `bash` command just above this

**Review by patriciasantaana — APPROVED** [review]

File: content/analytics/graphql-api/getting-started/execute-graphql-query.md (hunk 1 above)
Comment (victor-perov) [reply content/analytics/graphql-api/getting-started/execute-graphql-query.md:40]: Ah, interesting. Thanks for fixing it, @kodster28!

Comment (victor-perov) [reply content/analytics/graphql-api/getting-started/execute-graphql-query.md:40]: But why CI didn't notify about failed build?

Comment (pedrosousa) [reply content/analytics/graphql-api/getting-started/execute-graphql-query.md:40]: CI isn't using the exact same steps as our Cloudflare Pages deployment. I'm proposing a fix in #6754.

**Pull request merged**